docs/mindsdb-docs/docs/sql/create/predictor.md

```diff
@@ -21,35 +23,35 @@ Query OK, 0 rows affected (x.xxx sec)
-After you run the `#!sql CREATE PREDICTOR` statement, you can check the status of the training model by selecting from the `#!sql mindsdb.predictors`.
+!!! TIP "Checking Model Status"
+    After you run the `#!sql CREATE PREDICTOR` statement, you can check the status of the training process by querying the `#!sql mindsdb.predictors` table.
 
-```sql
-SELECT *
-FROM mindsdb.predictors
-WHERE name='[predictor_name]';
-```
+    ```sql
+    SELECT *
+    FROM mindsdb.predictors
+    WHERE name='[predictor_name]';
+    ```
 
-On execution, we get:
+    On execution, we get:
 
-```sql
-+------------------------+-----------------------------------+----------------------------------------+-----------------------+-------------+---------------+-----+-----------------+----------------+
-|name                    |status                             |accuracy                                |predict                |update_status|mindsdb_version|error|select_data_query|training_options|
-+------------------------+-----------------------------------+----------------------------------------+-----------------------+-------------+---------------+-----+-----------------+----------------+
-|predictor_name          |generating or training or complete |number depending on the accuracy metric |column_to_be_predicted |up_to_date   |22.7.5.0       |     |                 |                |
-+------------------------+-----------------------------------+----------------------------------------+-----------------------+-------------+---------------+-----+-----------------+----------------+
-```
+    ```sql
+    +------------------------+-----------------------------------+----------------------------------------+-----------------------+-------------+---------------+-----+-----------------+----------------+
+    |name                    |status                             |accuracy                                |predict                |update_status|mindsdb_version|error|select_data_query|training_options|
+    +------------------------+-----------------------------------+----------------------------------------+-----------------------+-------------+---------------+-----+-----------------+----------------+
+    |predictor_name          |generating or training or complete |number depending on the accuracy metric |column_to_be_predicted |up_to_date   |22.7.5.0       |     |                 |                |
+    +------------------------+-----------------------------------+----------------------------------------+-----------------------+-------------+---------------+-----+-----------------+----------------+
+    ```
 
 ## Example
 
-This example shows how you can train a Machine Learning model called home_rentals_model to predict the rental prices for real estate properties inside the dataset.
+This example shows how to create and train a machine learning model called `home_rentals_model` and predict the rental prices for real estate properties inside the dataset.
 
 ```sql
```

Comment:
Let delete both `AS {name}` statements.
First one (`AS rentals`) exists from times when we created new datasource for each such request. That was the name of that datasource. Currently this statement is meaningless.
Second (AS price) is not working at the moment, and i dont think it will be fixed in next month, so let delete it too.

Reply:
Deleted. For both examples.

Reply:
Also, I will make this change to all other queries in the docs to keep them up to date.